src/wl_window.c

```diff
@@ -605,6 +605,41 @@ void _glfwPlatformFocusWindow(_GLFWwindow* window)
                     "Wayland: Focusing a window requires user interaction");
 }
 
+void _glfwPlatformResizeWindow(_GLFWwindow* window, int border)
+{
+    int wlBorder;
+    switch (border)
+    {
+        case GLFW_WINDOW_LEFT:
+            wlBorder = WL_SHELL_SURFACE_RESIZE_LEFT;
+            break;
+        case GLFW_WINDOW_TOP:
+            wlBorder = WL_SHELL_SURFACE_RESIZE_TOP;
+            break;
+        case GLFW_WINDOW_RIGHT:
+            wlBorder = WL_SHELL_SURFACE_RESIZE_RIGHT;
+            break;
+        case GLFW_WINDOW_BOTTOM:
+            wlBorder = WL_SHELL_SURFACE_RESIZE_BOTTOM;
+            break;
+        case GLFW_WINDOW_TOPLEFT:
+            wlBorder = WL_SHELL_SURFACE_RESIZE_TOP_LEFT;
+            break;
+        case GLFW_WINDOW_TOPRIGHT:
+            wlBorder = WL_SHELL_SURFACE_RESIZE_TOP_RIGHT;
+            break;
+        case GLFW_WINDOW_BOTTOMLEFT:
+            wlBorder = WL_SHELL_SURFACE_RESIZE_BOTTOM_LEFT;
+            break;
+        case GLFW_WINDOW_BOTTOMRIGHT:
+            wlBorder = WL_SHELL_SURFACE_RESIZE_BOTTOM_RIGHT;
+    }
```

Comment:
The absence of a default case will make compilers yell at you for having `wlBorder` potentially uninitialised, you should either initialise it at declaration, or (better imo) add a default case which `assert(GLFW_FALSE)` to tell the compiler this can never happen.